workspace/utilities/lib/master-title.xsl

```diff
@@ -33,23 +33,37 @@
 	</xsl:template>
 
 <!-- Default master title template -->
-	<xsl:template name="default-master-title">
+<xsl:template name="default-master-title">
 		<xsl:variable name="cur-page-title">
 			<xsl:call-template name="page-title" />
 		</xsl:variable>
 		
-		<xsl:if test="string-length($cur-page-title) != 0 and count(/data/params/page-types/item[@handle = 'index']) = 0">
-			<xsl:value-of select="$cur-page-title"/>
-			<xsl:text> - </xsl:text>
-		</xsl:if>
-		
 		<xsl:choose>
-			<xsl:when test="string-length($site-name) != 0">
-				<xsl:value-of select="$site-name" />
+			<xsl:when test="string-length($page-metas/titre) != 0">
```

Comment:
What if the CMS is only in english and the handle `titre` is replaced by `title` ? I don't think we should use nodes directly from the data modeling but use a param with a default value of `titre` instead.

Reply:
@hotdoy @nitriques

Reply:
those are part of the templates and never change. But yeah we could.